config/api/src/main/java/org/jboss/windup/config/AbstractRuleProvider.java

```diff
@@ -131,7 +133,7 @@ public String toString()
         StringBuilder builder = new StringBuilder();
         builder.append(metadata.getID());
 
-        if (!metadata.getID().equals(metadata.getOrigin()))
+        if (this instanceof RulePhase || !metadata.getID().equals(metadata.getOrigin()))
```

Comment:
It skips printing of generated ID for phases, which is not useful output at all.